tests/queries/0_stateless/03362_into_outfile_atomic_truncate.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+function perform()
+{
+    local test_id=$1
+    local query=$2
+    local expected=$3
+    local is_compressed=$4
+
+    echo "performing test: $test_id"
+    ${CLICKHOUSE_CLIENT} --query "$query"
+    if [ "$?" -eq 0 ]; then
+        local actual
+        if [ "$is_compressed" = "1" ]; then
+            actual=$(gunzip -c "${CLICKHOUSE_TMP}/test_atomic_$test_id.out.gz")
```

Comment:
Nitpick: I think it's more clear to use curly braces for variables within a string, especially if there are several variables involved:

```suggestion
            actual=$(gunzip -c "${CLICKHOUSE_TMP}/test_atomic_${test_id}.out.gz")
```